navigation/src/main/java/com/graphhopper/navigation/NavigateResponseConverter.java

```diff
@@ -191,7 +309,8 @@ private static ObjectNode putInstruction(PointList points, InstructionList instr
 
         if (intersections.size() == 0) {
             // this is the fallback if we don't have any intersections.
-            // this can happen for via points or finish instructions or when no intersection details have been requested
+            // this can happen for via points or finish instructions or when no intersection
+            // details have been requested
```

Comment:
Actually, should we create this fallback intersection in the the filter method above? That way we can get rid of this special case here?

Reply:
It feels IMO not the right place in a filterXXX Method to actually create a dummy entry.  It will complicate the method even more